MELA/fortran/mod_Parameters.F90

```diff
@@ -610,6 +797,7 @@ MODULE ModParameters
 integer, public, target :: NuM_ = 15
 integer, public, target :: NuT_ = 16
 integer, public, target :: Hig_ = 25
+integer, public, target :: Hig2_= 49
```

Comment:
No need for Hig2, Hig is enough. If second resonance couplings are set, better to transfer them to first resonance.

Reply:
Also, "signal"=Hig+Hig2, and the only ways Hig or Hig2 are enabled are through gh or gh2 couplings and Mreso>=0 vs Mreso2>=0.